services/modrinth/modrinth-version.tester.js

```diff
@@ -0,0 +1,12 @@
+import { createServiceTester } from '../tester.js'
+import { isMetric } from '../test-validators.js'
+
+export const t = await createServiceTester()
+
+t.create('Version')
+  .get('/AANobbMI.json')
+  .expectBadge({ label: 'version', message: isMetric })
```

Comment:
The test for this is failing with

```
ValidationError: message mismatch: "value" with value "mc1.19.2-0.4.4" fails to match the required pattern: /^([1-9][0-9]*[kMGTPEZY]?|[1-9]\.[1-9][kMGTPEZY])$/
```

We do have a selection of test validators in https://github.com/badges/shields/blob/master/services/test-validators.js for validating version numbers, but I'm not sure `mc1.19.2-0.4.4` will match any of them. We might need a special validator for this. Are there any docs on what the format of a modrinth version should be?

Reply:
>Are there any docs on what the format of a modrinth version should be?

There aren't, it's up to the author to choose a version. I could choose another Modrinth project to run the test against but then if they change their versioning it will fail. Perhaps we could do a simpler match like `.*\d+\.\d+(\.d+)?.*`? It's not as strict, but less likely to break.

Reply:
Yeah something like that seems like a reasonable tradeoff.